src/net/db.rs

```diff
@@ -872,12 +1016,36 @@ impl PeerDB {
         Ok(allow_rows)
     }
 
+    /// Insert or replace stacker DBs for a peer, given its slot
+    pub fn insert_or_replace_stacker_dbs<'a>(
+        tx: &mut Transaction<'a>,
+        slot: u32,
+        smart_contracts: &[ContractId],
+    ) -> Result<(), db_error> {
+        for cid in smart_contracts {
+            debug!("Add Stacker DB to slot {}: {}", slot, cid);
```

Comment:
Can you consolidate this into a single debug line for the function call, rather than 1 per stacker db.

Reply:
This is now a `test_debug!`

Reply:
Fixed